analyzers/src/ITs.JsonParser/IssueParser.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.IO;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using ITs.JsonParser.Json;
+using ITs.JsonParser.Reports;
+
+namespace ITs.JsonParser;
+
+/// <summary>
+/// Reads SARIF reports from "output/" and writes custom reports to "actual/".
+/// </summary>
+public static class IssueParser
+{
+    private static readonly Regex RxSonarRule = new("^S[0-9]+$", RegexOptions.Compiled);
+    private static readonly JsonSerializerOptions SerializerOptions = new() { PropertyNameCaseInsensitive = true, WriteIndented = true };
+
+    public static void Execute(string inputRootPath, string outputRootPath)
```

Comment:
We don't create any sub-paths in this method, so we can simplify the names
```suggestion
    public static void Execute(string inputPath, string outputPath)
```